tests/unit/select_test.coffee

```diff
@@ -258,3 +258,70 @@ test "Show no-result message if has content but filtered content is empty", ->
       'Empty content block not displayed'
     ok isPresent(noResultSelector, selectElement),
       '"No result" message displayed'
+
+test 'optionValuePath with POJOs', ->
+  expect 1
+
+  obj1 = {name: 'reddit', value: 1}
+  obj2 = {name: 'red', value: 2}
+  data = [obj1, obj2]
```

Comment:
no. This test was initially written to reproduce the problem, before I understood what the problem is.

But that's a good point and we should test that.